io.github.alainm23.planify.json

```diff
@@ -34,7 +34,66 @@
                 {
                     "type": "git",
                     "url": "https://github.com/elementary/granite.git",
-                    "tag": "7.2.0"
+                    "tag": "7.6.0",
```

Comment:
I think this will no longer be necessary, Granite will be removed in the next version of Planify.

https://github.com/alainm23/planify/pull/1681